example/.ruby-version

```diff
@@ -1 +0,0 @@
-2.7.5
```

Comment:
Why did we remove this?

Reply:
Following a migration tool to update RN at `0.70.14` the .ruby-version was marked as removed. I have restore this file here https://github.com/pagopa/io-react-native-crypto/pull/27/commits/5ef0dac32617817fda0e612e7b670701ef8727f5